Complete: Savan: Task 793-799: Bianet, Imppres datasets

mirror3 opened the pull request.

swarooprm commented:
@SavanDoshi, Wondering if you can help review these tasks?

SavanDoshi commented:
There are many ASCI codes in all Jason file to remove that use "json.dumps(your_json_string, indent=4, ensure_ascii=False)" to generate Jason file.

Try to explain why the second sentence is a translation of the first sentence.

There are no proper explanations for the **translation and classification task**. Please refer to the following link where you can find the proper definition and explanation. (https://github.com/allenai/natural-instructions-expansion/pull/233).

- Update definition and explanations.
- Also, in the explanation whenever you are referring to a particular word from input or output use "word" to mention the word.

**task798:**
- Change the definition write like: "In this task, you are given premise and hypothesis. Your task is to classify them among three classes \"Positive\", \"Negative\", and \"Neutral\" according to a relationship of premise and hypothesis."
- In the positive and negative examples you have used "negated" instead of negative. update this also.
- Some of the Instances have output: "Not_In_Example" what does that mean??. As I understand this is a classification task therefore output must be among (Positive, Negative, and Neutral ).


**task799:**
- Update the definition and explain it clearly.
- This task's category is "Text Generation" but in the json file it is"Classification" this must be error.
- Check all the jason files for grammar mistakes.

mirror3 commented:
Hi @SavanDoshi , I have updated the files as per all the comments mentioned. The only difference is in task 798 where I have used "negated" instead of "negative" because that is how "presupposition" is mentioned in the dataset given. I have updated the definition according to this.

Kindly check and let me know if everything is alright.

swarooprm commented:
@SavanDoshi, check and let me know if you are happy with the PR or have any other feedback?

SavanDoshi commented:
@swarooprm Looks good to me. He has addressed all the comments.

swarooprm commented:
Your HW2 is complete now.

Would you like to submit the same tasks to our big project repo https://github.com/allenai/natural-instructions-expansion/pulls? You just need to fork that repo, rename tasks starting from the latest number in the repo and create PR there. This way, you can be part of our research on "learning from instructions instead of learning from examples (conventional machine learning)". This is purely optional and your HW2 grade does not depend on this @mirror3

swarooprm commented:
@mirror3, Just a reminder that, if you plan to submit these tasks to the main Github repo and join our research project, Monday (27th Sept) is the deadline. Again, this is optional and independent of your HW2 grade.
